GW2EIEvtcParser/EIData/CombatReplay/Decorations/Forms/DoughnutDecoration.cs

```diff
@@ -1,74 +1,71 @@
-﻿using System;
-using System.Collections.Generic;
-using GW2EIEvtcParser.ParsedData;
+﻿using GW2EIEvtcParser.ParsedData;
 
-namespace GW2EIEvtcParser.EIData
+namespace GW2EIEvtcParser.EIData;
+
+public class DoughnutDecoration : FormDecoration
```

Comment:
Certain decorations became public, while others remain internal, what is the motivation behind that change?

Reply:
That must be a leftover from testing, before I added the assembly attribute to share internals.
Ill address it shortly

Reply:
On second thought, it might also have to do with serialization.
Using the more efficient compiletime-generated serializer requires explicit references to superclasses that may be encountered during serialization, and those superclasses as well as the base class must all share the same visibility

Reply:
Yea so its the serialization.
Do you by chance have an example of decorations that are not public? I assume I forgot to add the explicit marker for those classes.

Reply:
Decorations/Images/BackgroundIconDecoration.cs is the only one remaining from what I see.
In that case, perhaps constructors of classes that became public should become internals?

Reply:
Wait, we only serialize the DecorationDescription classes though, not the Decoration classes.

Reply:
I think you are correct.
I removed the attributes and changed concrete decorations back to in internal visibility